src/Multipart/UploadState.php

```diff
@@ -76,6 +91,47 @@ public function setPartSize($partSize)
         $this->partSize = $partSize;
     }
 
+    /**
+     * Sets the 1/8th thresholds array. $totalSize is only sent if
+     * 'track_upload' is true.
+     *
+     * @param $totalSize numeric Size of object to upload.
+     *
+     * @return array
+     */
+    public function setProgressThresholds($totalSize)
```

Comment:
Is there a reason these two functions are public instead of private?

Reply:
They're UploadState methods that are accessed by MultipartUploader and MultipartCopy